pkgs/applications/networking/sync/rclone/default.nix

```diff
@@ -19,6 +19,9 @@ buildGoPackage rec {
 
   postInstall = ''
     install -D -m644 $src/rclone.1 $man/share/man/man1/rclone.1
+    mkdir -p $bin/share/zsh/site-functions $bin/share/bash-completion/completions/
+    $bin/bin/rclone genautocomplete zsh $bin/share/zsh/site-functions/_rclone
+    $bin/bin/rclone genautocomplete bash $bin/share/bash-completion/completions/rclone.bash
```

Comment:
if you want to use `installShellCompletion`:

```
nativeBuildIinputs = [ installShellCompletion ];
```
```
$bin/bin/rclone genautocomplete zsh > _rclone
$bin/bin/rclone genautocomplete bash > rclone.bash
installShellCompletion _rclone rclone.bash
```

Reply:
unfortunately it does not work with `buildGoPackage`

Reply:
:(